Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8-slim-buster as base
+
+WORKDIR /app
+COPY poetry.lock pyproject.toml /app/
+
+RUN pip install poetry && poetry install --no-root && poetry add gunicorn
```

Comment:
Really minor, but it's nice to install poetry without depending on pip at all, e.g. following the instructions on [their website](https://python-poetry.org/docs/)

Separately, if we always need to add gunicorn here, it might make more sense to add it directly to our project outside docker so that it's picked up by default.

Reply:
changed as advised.